getting-started/emails.mdx

```diff
@@ -0,0 +1,59 @@
+---
+title: "Transactional Emails"
+description: "We send emails to your users for common actions."
+---
+
+For example:
+
+- Passwordless logins
+- Reset password emails
+- Organization invitations
+
+These emails will share the same color scheme and style you configure for your
+hosted authentication pages. We manage these emails for you by default, so you don't need
+to connect an external provider until you're ready.
+
+## Sending Emails From Your Domain
+
+By default, transactional emails will be sent from welcome@propelauth.dev; however, 
+you can set up your projects to send emails from a custom email address and domain, 
+which can help with email deliverability.
+
+First, go to project settings. If you haven't [created a verified production domain](/getting-started/deploy-to-prod), 
+you will see a link to the Go Live portion of your dashboard in the Send Emails 
+From Your Domain section.
+
+![configure your production domain first](/images/emails-1.png)
+
+Once you have a verified production domain, click the Configure button.
+
+![project settings](/images/emails-2.png)
+
+Fill out your desired Sender Name and email address in the following modal. 
+The Reply-To email will default to the same address as the From email, but this 
+can be overwritten if desired. One thing to note: your email domains must match 
+the verified production domain you set up in the Go Live section of your dashboard.
+
+![email customization](/images/emails-3.png)
+
+Once you have customized your email domains and hit Save, you will be returned to 
+the dashboard's Project Settings section. Here, click the Verify DNS Records button.
+
+![project settings](/images/emails-4.png)
+
+Use the values in the modal that appears here in the DNS Server of your choice. 
+Instructions for how to do this will vary depending on where you purchased the 
+domain from. If you need any help, please reach out to us at support@propelauth.com. 
+It may take a few minutes for the DNS records to be set and verified.
+
+![verify dns records](/images/emails-5.png)
+
+Once your domain has been verified, emails will now be sent from the domains you 
+specified. If you would like to change these emails in the future, you just need 
+to update the email configuration in Project Settings, and verify once again with 
```

Comment:
Most changes actually won't require them to change their DNS settings. If they are changing from blah@example.com to blah2@example.com, it happens automatically.

The error messaging here is pretty good, so you can just probably end after update the email configuration in Project Settings.